app/src/main/java/com/example/myapplication/Mana.java

```diff
@@ -8,228 +8,231 @@
-  @Override
-  public String toString() {
-    String mana_string = ordered_by + ": " + getTypeStr(type) + ", "
-        + paymentStr(payment_method);
-    if(in_order == 0){
-      mana_string += ", Ready for ordering: " + isReadyToOrder() + ".";
-    } else{
-      mana_string += ", Part of order serial number: " + in_order;
-    }
-    if(costumer_notes != null)
-      mana_string += ". NOTE: " + costumer_notes;
-    return mana_string + " (Mana serial number " + serial_no + ")";
-  }
-
-  // Getters
-  int getSerial(){
-    return serial_no;
-  }
-
-  private int getType(){
-    return type;
-  }
-
-  /**
-   * returns the type of the Mana represented as a String
-   * @param type the type of the Mana (int)
-   * @return String representation of the type of the Mana
-   */
-  private static String getTypeStr(int type){
-    switch(type){
-      case PITA:
-        return "Pita";
-      case LAFA:
-        return "Lafa";
-      case HALF_PITA:
-        return "Half Pita";
-      case HALF_LAFA:
-        return "Half Lafa";
-      default:
-        return "Type not set";
-    }
-  }
-
-  int getPrice(){
-    return priceByType(type);
-  }
-
-  /**
-   * Gets the payment method represeneted as a String
-   * @param payment_method the payment method for the Mana, an int
-   * @return a String representation of the payment method associated with the Mana
-   */
-  private String paymentStr(int payment_method) {
-    switch(payment_method){
-      case MEZUMAN:
-        return "Mezuman";
-      case CREDIT:
-        return "Credit";
-      case APP:
-        return "Applicatio";
-      default:
-        return "ooops";
-    }
-  }
-
-  /**
-   * A static method that stores the information of the prices for each Mana
-   * @param type the type of the Mana, as an int
-   * @return the price associated with the type, an int
-   */
-  private static int priceByType(int type){
-    switch(type){
-      case PITA:
-        return PITA_PRICE;
-      case LAFA:
-        return LAFA_PRICE;
-      case HALF_PITA:
-        return HALF_PITA_PRICE;
-      case HALF_LAFA:
-        return HALF_LAFA_PRICE;
-      default:
-        System.out.println("ooops2");
-        return 0;
-    }
-  }
-
-  // Setters
-  boolean setType(int new_type){
-    if(status == OPEN){
-      type = new_type;
-      price = getPrice();
-      return true;
-    }
-    return false;
-  }
-
-  boolean setPaymentMethod(int new_method){
-    if(status == OPEN){
-      payment_method = new_method;
-      return true;
-    }
-    return false;
-  }
-
-  boolean addNote(String note){
-    if(status == OPEN){
-      costumer_notes = note;
-      return true;
-    }
-    return false;
-  }
-
-  boolean isReadyToOrder(){
-    return (type != INVALID) && (payment_method != INVALID);
-  }
-
-  void lock() {
-    status = LOCKED;
-  }
-
-  void addToOrder(int order_no){
-    in_order = order_no;
-  }
-
-  int inOrder() {
-    return in_order;
-  }
-}
+    // Constants
+    //-- Types of manot
```

Comment:
I think it will be better to make each type of constant an enum.
but since this is avner's code, it shouldn't be your job. maybe I will do it after we merge into Deveplop